### PR TITLE
fix: Transilation

### PR DESCRIPTION
**Pull request by sabesansathananthan**

### CODE REVIEW

**Comment by sabesansathananthan** [comment]
@esau-morais Could you please review it

File: translations/README.te.md [hunk 1]
```diff
@@ -81,4 +85,3 @@ cd React-GitHub-Resume
 ## 📄 లైసెన్స్
 
 ఈ ప్రొజెక్ట్ యం.ఐ.టి లైసెన్స్ కింద ఉన్నది. మరిన్ని విషయాల కోసం [లైసెన్స్](./LICENSE) ఫైలు చూడండి.
```
Comment (esau-morais) [inline translations/README.te.md:87]: Please, correct the `LICENSE` route to `../LICENSE`

File: translations/README.it.md [hunk 2]
```diff
@@ -82,4 +82,4 @@ cd React-GitHub-Resume
 
 ## 📄 Licenza
 
-Questo progetto è sotto la lincenza MIT - leggi il file [LICENSE](./LICENSE) per ulteriori dettagli
+Questo progetto è sotto la lincenza MIT - leggi il file [Licenza](./LICENSE) per ulteriori dettagli
```
Comment (esau-morais) [inline translations/README.it.md:85]: Please, correct the `LICENSE` route to `../LICENSE`

File: translations/README.cs.md [hunk 3]
```diff
@@ -46,38 +46,42 @@ cd React-GitHub-Resume
...
 4. Přejdi na adresu <http://localhost:3000>
 
 ## [Živé demo](https://react-github-resume.vercel.app/)
 
 [![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https://github.com/sabesansathananthan/React-GitHub-Resume)
 
-## 📄 Licence
+## 📄 licenci
 
 Tento projekt spadá pod licenci MIT — pro více detailů navštiv soubor s [LICENCÍ](./LICENSE)
```
Comment (esau-morais) [inline translations/README.cs.md:87]: Please, correct the `LICENSE` route to `../LICENSE`

**Review by esau-morais — CHANGES_REQUESTED** [review]
Thank you for contribute correcting the translation files!

_Note: Just see the requests to be approved_

File: translations/README.cs.md [hunk 4]
```diff
@@ -4,14 +4,14 @@
 
 ## Seznam obsahu
 
-- [O projektu](#o%20projektu)
-- [Použité nástroje](#pou%C5%BEit%C3%A9%20n%C3%A1stroje)
+- [O projektu](#o-projektu)
+- [Použité nástroje](#Použité-nástroje)
 - [Instalace](#instalace)
-- [Demo](#%C5%BEiv%C3%A9%20demo)
+- [Demo](#Živé-demo)
```
Comment (esau-morais) [inline translations/README.cs.md:10]: Probably, when you start a link with uppercase, it'll not be more a link. Try to change:

- [ ] `P` to `p`
- [ ] `Ž` to `ž`

File: translations/README.it.md [hunk 5]
```diff
@@ -4,14 +4,14 @@
 
 ## Indice
 
-- [Informazioni](#about)
-- [Materiale utilizzato](#built-with)
-- [Installazione](#installation)
-- [Demo](#live-demo)
+- [Informazioni](#Informazioni)
```
Comment (esau-morais) [inline translations/README.it.md:7]: Probably, when you start a link with uppercase, it'll not be more a link. Try to change:

- [ ] `I` to `i`
- [ ] `M` to `m`
- [ ] `I` to `i`
- [ ] `D` to `d`

File: translations/README.ru.md [hunk 6]
```diff
@@ -4,14 +4,14 @@
 
 ## Таблица содержимого
 
-- [О проекте](#about)
-- [Создано с помощью](#built-with)
-- [Установка](#installation)
-- [Демо](#live-demo)
+- [О проекте](#О-проекте)
```
Comment (esau-morais) [inline translations/README.ru.md:7]: Probably, when you start a link with uppercase, it'll not be more a link. Try to change:

- [ ] `О` to `о` 
- [ ] `С` to `с`
- [ ] `У` to `у`
- [ ] `Д` to `д`

File: translations/README.te.md (hunk 1 above)
Comment (sabesansathananthan) [reply translations/README.te.md]: fixed

File: translations/README.cs.md (hunk 3 above)
Comment (sabesansathananthan) [reply translations/README.cs.md]: fixed

File: translations/README.it.md (hunk 2 above)
Comment (sabesansathananthan) [reply translations/README.it.md]: fixed

File: translations/README.ua.md [hunk 7]
```diff
@@ -4,14 +4,14 @@
 
 ## Таблиця вмісту
 
-- [Про проект](#about)
-- [Створено за допомогою](#built-with)
-- [Встановлення](#installation)
-- [Демо](#live-demo)
+- [Про проект](#Про-проект)
```
Comment (esau-morais) [inline translations/README.ua.md:7]: Probably, when you start a link with uppercase, it'll not be more a link. Try to change:

- [ ] `П` to `п`
- [ ] `С` to `с`
- [ ] `В` to `в`
- [ ] `Д` to `д`

**Review by esau-morais — COMMENTED** [review]
Please, test these requests before

_Note: Use [markdownlint](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) to preview the README before commit and test if the links are working_

File: translations/README.cs.md (hunk 4 above)
Comment (sabesansathananthan) [reply translations/README.cs.md]: fixed

File: translations/README.it.md (hunk 5 above)
Comment (sabesansathananthan) [reply translations/README.it.md]: fixed

File: translations/README.ru.md (hunk 6 above)
Comment (sabesansathananthan) [reply translations/README.ru.md]: fixed

File: translations/README.ua.md (hunk 7 above)
Comment (sabesansathananthan) [reply translations/README.ua.md]: fixed

**Comment by sabesansathananthan** [comment]
@esau-morais I think I fixed all the errors. but could you please review it

**Comment by esau-morais** [comment]
> @esau-morais I think I fixed all the errors. but could you please review it

Thank you for make the changes.

_Note: There are some links that are not working when I link (probably it's the alphabet)_

**Comment by sabesansathananthan** [comment]
> > @esau-morais I think I fixed all the errors. but could you please review it
> 
> Thank you for make the changes.
> 
> _Note: There are some links that are not working when I click (probably it's the alphabet)_

I also face the same issue as well. only the installation link is work for all languages

**Comment by esau-morais** [comment]
> > > @esau-morais I think I fixed all the errors. but could you please review it
> > 
> > 
> > Thank you for make the changes.
> > _Note: There are some links that are not working when I click (probably it's the alphabet)_
> 
> I also face the same issue as well. only the installation link is work for all languages

The link stay with the problem. 

_Note: Probably the only way is adding the all host to the topic_

**Comment by sabesansathananthan** [comment]
Now I will merge the changes. Will fix it in the next PR. I will change one by one using inspect in the browser.

**Pull request merged**